Library/Homebrew/services/cli.rb

```diff
@@ -11,6 +11,11 @@ module Cli
       extend FileUtils
       extend Utils::Output::Mixin
 
+      sig { void }
+      def self.initialize
```

Comment:
This is a bit weird, when is it called? If this exists just to define the `@sudo_service_user` then consider doing just at first use.